.github/workflows/build.yml

```
@@ -180,14 +180,17 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
-            github.rest.repos.createCommitStatus({
+            github.rest.checks.create({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              sha: '${{ github.event.pull_request.head.sha }}',
-              state: 'success',
-              context: 'Deploy preview',
-              description: 'Live preview of the current branch',
-              target_url: 'https://itwin.github.io/iTwinUI/${{ github.event.number }}',
+              head_sha: '${{ github.event.pull_request.head.sha }}',
+              conclusion: 'neutral',
+              name: 'deploy-preview',
+              output: {
+                title: 'Deploy preview',
+                summary: 'Live preview of the current branch',
+                text: '[Portal](https://itwin.github.io/iTwinUI/${{ github.event.number }})\n[CSS](https://itwin.github.io/iTwinUI/${{ github.event.number }}/css)\n[React](https://itwin.github.io/iTwinUI/${{ github.event.number }}/react)'
```

This doesn't need to be written in one line, you can use line-breaks. And since it's markdown, you can make it a list.

Other things:
1. we don't care about `portal` anymore. This check *is* the portal.
2. we can add `website` preview here
3. (in a future PR) we can add hosted versions of playgrounds and e2e test

```suggestion
                text: `\
            - [css-workshop](https://itwin.github.io/iTwinUI/${{ github.event.number }}/css)
            - [react-workshop](https://itwin.github.io/iTwinUI/${{ github.event.number }}/react)
            - [docs website](https://deploy-preview-${{ github.event.number }}--itwinui-previews.netlify.app/)`
```